notebook/static/tree/js/main.js

```diff
@@ -209,7 +209,7 @@ requirejs([
     
     // load tab if url hash
     if (window.location.hash) {
-        $("#tabs").find("a[href=" + window.location.hash + "]").click();
```

Comment:
Thanks! It looks like it's possible to use `'single quotes'` ((jQuery docs)[https://api.jquery.com/attribute-equals-selector/]) rather than escaping double quotes, which would be a bit easier to read, IMO. It's not a big deal, though.

Reply:
Good point, much more readable. Fixed!